ChangeLog.md

```diff
@@ -10,13 +10,20 @@
 1. `--log-level=none` now drops no logs, and has a listing in `--help`.
 1. Fixed bug where piping was not picking up the service version override, making it not work well against Azure Stack.
 1. Fixed a timeout when uploading particularly large files to ADLSG2.
+1. Fixed single wildcard match uploads
```

Comment:
Missing two things: Fractional block size and bandwidth cap.

Reply:
Line 23 and 25